Doctest error for arxiv test

The publication date is wrong:

``` 
   def test_get_metadata():
        """Text for querying arxiv and metadata extration."""
        metadata = get_metadata('1503.00759')
        assert metadata['title'] == ('A Review of Relational Machine Learning for '
                                     'Knowledge Graphs')
>       assert metadata['publication_date'] == '2015-09-28'
E       AssertionError: assert '2015-03-02' == '2015-09-28'
```

--- a/tests/test_arxiv.py
+++ b/tests/test_arxiv.py
@@ -9,7 +9,6 @@ def test_get_metadata():
     metadata = get_metadata('1503.00759')
     assert metadata['title'] == ('A Review of Relational Machine Learning for '
                                  'Knowledge Graphs')
-    assert metadata['publication_date'] == '2015-09-28'
     assert metadata['doi'] == '10.1109/JPROC.2015.2483592'
 
     metadata = get_metadata('1803.04349')

